eva_cttv_pipeline/consequence_type.py

```diff
@@ -22,12 +22,11 @@ def process_consequence_type_file_tsv(snp_2_gene_filepath):
             line_list = line.split("\t")
 
             variant_id = line_list[0]
-            ensembl_gene_id = line_list[1]
-            so_term = line_list[3]
+            ensembl_gene_ids = line_list[1].split(",")
+            so_terms = line_list[3].split(",")
 
-            ensembl_gene_ids = ensembl_gene_id.split(",")
             for ensembl_gene_id in ensembl_gene_ids:
```

Comment:
Wouldn't it be best to remove the loop and insert all the genes and SO terms at once?